### PR TITLE
Upgrade decidim 0.16.0

### PR DESCRIPTION
**Pull request by aitorlb**
- Update `decidim` version `'0.16.0'`
- Run migrations
- Update tag:
  - gem `decidim-verifications-salou_census`, tag: `'0.3.2'`
- Add upgrade notes
- Add `decidim:open_data:export` to `config/schedule.rb`

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -15,15 +15,11 @@ gem 'openssl'
 gem 'figaro', '>= 1.1.1'
 
 gem 'decidim', DECIDIM_VERSION
-gem 'decidim-verifications-salou_census', git: "git@gitlab.coditdev.net:decidim/decidim-verifications-salou_census.git", tag: "v0.3.1"
+gem 'decidim-verifications-salou_census', path: '../'
```
Comment (tramuntanal) [inline Gemfile]: I think you commited this modification from development.

**Review by tramuntanal — CHANGES_REQUESTED** [review]
great! but be aware of changes in the Gemfile

File: Gemfile (hunk 1 above)
Comment (aitorlb) [reply Gemfile]: Yes. I'm waiting for [this](https://gitlab.coditdev.net/decidim/decidim-verifications-salou_census/merge_requests/3) to be merged to update de Gemfile

**Review by isaacmg410 — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (aitorlb) [reply Gemfile]: @tramuntanal can you resolve this, please? I can't merge

**Review by tramuntanal — APPROVED** [review]

**Pull request merged**